tools/logger/convert.c

```diff
@@ -279,18 +279,35 @@ static double to_usecs(uint64_t time)
 	return (double)time / global_config->clock;
 }
 
+/** Justified timestamp width for printf format string */
+static unsigned int timestamp_width(unsigned int precision)
+{
+	/* 64bits yields less than 20 digits precision. As reported by
+	 * gcc 9.3, this avoids a very long precision causing snprintf()
+	 * to truncate time_fmt
+	 */
+	assert(precision >= 0 && precision < 20);
+	/*
+	 * 12 digits for units is enough for 1M seconds = 11 days which
+	 * should be enough for most test runs.
+	 *
+	 * Add 1 for the comma when there is one.
+	 */
+	return 12 + (precision > 0 ? 1 : 0) + precision;
+}
+
```

Comment:
Previous version is capable to skip timestamp printing to allow easily use diff tool on logs, this commit removes this possibility

Reply:
Great catch, thanks @ktrzcinx for the review. Fixed in  6c1a4cd.

PS: any idea why QB never tested 6c8333d ? (PTS-31119)

Reply:
QB fails because of linking error in build with codec adapter: 
`src/audio/codec_adapter/codec/cadence.c.o:(.data+0x2c): undefined reference to "cadence_api_function"`

So it looks unrelated to this PR